ext/standard/tests/file/005_variation.phpt

```diff
@@ -52,12 +51,12 @@ $file_path = __DIR__;
 $file_handle = fopen("$file_path/005_variation1.tmp", "w");
 fclose($file_handle);
 stat_fn("$file_path/005_variation1.tmp");
-sleep(2);
+sleep(1);
```

Comment:
Feel free to delete this test entirely. Without checks for == / != this test is useless, and we already have other tests that do perform the necessary checks. As such, this test is just a waste of CI time :)